### PR TITLE
Add isAutoSelected

### PR DESCRIPTION
**Pull request by npm1**
Fixes https://github.com/fedidcg/FedCM/issues/497

***
<a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/pull/500.html" title="Last updated on Jan 17, 2024, 4:07 PM UTC (a86d228)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/500/b3674e5...a86d228.html" title="Last updated on Jan 17, 2024, 4:07 PM UTC (a86d228)">Diff</a>

### CODE REVIEW

**Review by yi-gu — APPROVED** [review]
LGTM. Thanks!

File: spec/index.bs [hunk 1]
```diff
@@ -375,6 +376,9 @@ This specification introduces a new type of {{Credential}}, called an {{Identity
     :   <b>{{IdentityCredential/token}}</b>
     ::  The {{IdentityCredential/token}}'s attribute getter returns the value it is set to.
         It represents the minted {{IdentityProviderToken/token}} provided by the [=IDP=].
+    :   <b>{{IdentityCredential/isAccountAutoSelected}}'s attribute getter returns the value it is
+        set to. It represents whether the user account was automatically selected when going through
+        the UI flow which resulting in this {{IdentityCredential}}.
```
Comment (cbiesinger) [inline spec/index.bs:381]: resulting -> resulted?

alternatively, s/which// ?

File: spec/index.bs [hunk 2]
```diff
@@ -366,6 +366,7 @@ This specification introduces a new type of {{Credential}}, called an {{Identity
   [Exposed=Window, SecureContext]
   interface IdentityCredential : Credential {
     readonly attribute USVString? token;
+    readonly attribute boolean isIdentityCredentialAutoSelected;
```
Comment (samuelgoto) [inline spec/index.bs:369]:
```suggestion
    readonly attribute boolean isAutoSelected;
```

Isn't the IdentityCredential prefix unneccesary here?

Comment (npm1) [reply spec/index.bs]: Renamed

**Review by samuelgoto — APPROVED** [review]
LGTM

**Review by bvandersloot-mozilla — COMMENTED** [review]
r+

**Comment by samuelgoto** [comment]
Thanks @bvandersloot-mozilla ! Merging!

**Pull request merged**